components/general/using_components.rst

```diff
@@ -35,24 +35,33 @@ whatever component you want.
 Once Composer has downloaded the component(s), all you need to do is include
 the ``vendor/autoload.php`` file that was generated by Composer. This file
 takes care of autoloading all of the libraries so that you can use them
-immediately::
-    // File example: src/script.php
+immediately:
+
+.. code:: php
+
+    // Sample script.php file.
+
     <?php
 
-    // update this to the path to the "vendor/"
-    // directory, relative to this file
```

Comment:
It just works if you follow docs and some may found it confusing as if they need to do something here, they don't. :) What do you think?